### PR TITLE
Linkage maps and QTLs data tables

### PR DESCRIPTION
**Pull request by Nuanda**
Adds two other tabs to the data tables view. The QTL query is "wrong" until Piotr performs the curation mentioned in #136 .

### CODE REVIEW

**Comment by Nuanda** [comment]
bump

File: app/models/concerns/pluckable.rb [hunk 1]
```diff
@@ -16,8 +16,9 @@ def self.pluck_columns
       columns = table_columns + ref_columns
       columns.each do |column|
         relation = column.to_s.split('.')[0].pluralize if column.to_s.include? '.'
-        next unless relation and relation != self.table_name
+        next unless relation && relation != self.table_name
```
Comment (nowakowski) [inline app/models/concerns/pluckable.rb]: I use `and`. Is this a best practice?

File: app/models/linkage_map.rb [hunk 2]
```diff
@@ -11,5 +11,24 @@ class LinkageMap < ActiveRecord::Base
...
 
+  default_scope { includes(plant_population: :taxonomy_term) }
+
+  include Pluckable
+
+  def self.table_data(params = nil)
+    pluck_columns
+  end
+
+  def self.table_columns
```
Comment (nowakowski) [inline app/models/linkage_map.rb]: Out of curiosity - why these columns? Seems to be plenty more payload in `linkage_maps`.

File: app/views/data_tables/index.html.haml [hunk 3]
```diff
@@ -11,5 +11,10 @@
       - table_column = model_param + '.' + table_column unless table_column.include? '.'
       %th= t("tables.#{table_column}")
 
+  - if model_class.respond_to? :count_columns
+    - model_class.count_columns.each do |count_column|
+      - count_column = count_column.scan(/\(([^\)]+)\)/).last.first
```
Comment (nowakowski) [inline app/views/data_tables/index.html.haml]: Bit puzzled. What is the intended behavior here?

File: app/models/concerns/pluckable.rb (hunk 1 above)
Comment (Nuanda) [inline app/models/concerns/pluckable.rb]: Yes, most of the time && is a better choice (these are slightly different operators) but if you prefer and and it works, it's ok with me.

File: app/models/linkage_map.rb (hunk 2 above)
Comment (Nuanda) [inline app/models/linkage_map.rb]: These are from the original Interstore CS table.

File: app/views/data_tables/index.html.haml (hunk 3 above)
Comment (Nuanda) [inline app/views/data_tables/index.html.haml]: To get only the content inside parentheses (). So from count(a.b) only a.b is taken, in order to deliver a proper join.

**Comment by nowakowski** [comment]
:+1:

**Pull request merged**